### PR TITLE
add magvel as a derived quantity

### PR DESCRIPTION
**Pull request by asalmgren**

### CODE REVIEW

File: src/derive/incflo_derive.cpp [hunk 1]
```diff
@@ -143,6 +143,61 @@ Real incflo::ComputeKineticEnergy ()
 }
 
 #if (AMREX_SPACEDIM == 2)
+void incflo::ComputeMagVel (int lev, Real /*time*/, MultiFab& magvel, MultiFab const& vel)
+{
+    BL_PROFILE("incflo::ComputeMagVel");
+
```
Comment (cgilet) [inline src/derive/incflo_derive.cpp:149]: I think `amrex::ignore_unused(lev)` might be useful here

**Review by cgilet — APPROVED** [review]

File: src/incflo.H [hunk 2]
```diff
@@ -162,6 +162,9 @@ public:
     //
     ///////////////////////////////////////////////////////////////////////////
 
+#ifdef USE_EB
```
Comment (cgilet) [inline src/incflo.H]: I think this should be `#ifndef AMREX_USE_EB`, right?

Comment (cgilet) [reply src/incflo.H]: sorry for my bad pseudo code

**Comment by asalmgren** [comment]
This will be replaced by a new PR

**Pull request closed without merging**